### PR TITLE
add reducers section with constant actions

### PR DESCRIPTION
**Pull request by shivamSspirit**
add reducers and their sections

### CODE REVIEW

File: src/constant/actions.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// export const val  = (function ALlActions(){console.log('hello')})()
+// val()
```
Comment (HeyNitin) [inline src/constant/actions.js:2]: remove commented code

File: src/reducers/watchLater.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        
+            // if (state?.watchLaterproducts?.find(item => item?.id === action.payload.id)) {
+                return {
+                    ...state,
+                    watchLaterproducts: action.payload,
+                    watchLaterCount:action?.payload?.length
+                }
+            // } else {
+            //     return {
+            //         ...state,
```
Comment (HeyNitin) [inline src/reducers/watchLater.js:19]: remove this

**Review by HeyNitin — APPROVED** [review]
you can merge after removing commented code

File: src/reducers/playlistmanagment.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
+export const playlistInitialState = {
+    playlistproducts: [],
+    playlistCount: 0,
+    currentselectedVideo: undefined,
+    videoCountinSinglePlayList: 0,
+}
+
+const PlaylistListReducer = (state = playlistInitialState, action) => {
+    //  console.log(action)
```
Comment (Rohitprasad83) [inline src/reducers/playlistmanagment.js:9]: remove unwanted code

**Review by Rohitprasad83 — APPROVED** [review]
- Good Naming conventions used
- Code is easily readable.
- remove the unwanted code.

**Pull request merged**